### DISCUSSION TITLE
polar alignment question

### DISCUSSION THREAD

**Discussion by airscottdenning** [mworion/MountWizzard4#227] [Q&A] [answered]
When performing a Polar Alignment with the 10Micron keypad, the final press of the Enter key completes the process. My understanding is that this also updates the current mount model to indicate perfect polar alignment.

Last night I used the Polar Alignment function in MW4 for the first time, but I don't understand how to mark the alignment "complete" in a way that's equivalent to that final "Enter" on the keypad that confirms that alignment is finalized.

I guess in practice no matter which way I polar align (keypad or MW4) I still need to build a new model afterward, so maybe there's no reason to mark the alignment as complete.

Could you explain the step-by-step process for PA in MW4 without using the keypad? Or are keypad actions an implied part of the process?

**Comment by Ruediger3** [comment, marked as answer]
Hi Scott,
the model does not get updated. You have to delete the existing and make a new one. PA voids the existing model. 

CS
Rüdiger

**Reply by mworion** [reply]
Hi Scott,
Rüdiger is right. Actually what happens in mount computer is that DAT is switched on again (if it was auto disabled in FW > 3.x) and the model is deleted afterwards.
If it would help, I also could enforce this workflow in MW4. Right now I avoid it at convenience and flexibility of the user.
Michel

**Comment by airscottdenning** [comment]
Thank you! I suggest the documentation might be updated to reflect these nuances. It's pretty clear what's going on when I use the keypad to do PA, but the GUI in MW4 is pretty uninformative about this. Maybe after clicking on a PA star the UI should activate the imaging panel and then prompt for "done" or "confirm" when the user is finished adjusting knobs. And then maybe a popup explaining that the model is now invalid would be helpful.

**Reply by mworion** [reply]
Hi Scott,
good ideas, I try to improve this part.
Michel
